extensions/oplogPopulator/OplogPopulator.js

```diff
@@ -231,6 +233,7 @@ class OplogPopulator {
             pipeline: changeStreamPipeline,
             handler: this._handleChangeStreamChangeEvent.bind(this),
             throwOnError: false,
+            useStartAfter: !this._mongoVersion.startsWith("4.0.")
```

Comment:
I am not sure this will work with the current BB CI, can we do this a bit more "semver"-like?

https://github.com/scality/backbeat/blob/development/8.6/.github/workflows/tests.yaml#L79

Reply:
Also, I am not sure the version is good enough. In case of upgrades, unless we explicitely set the featureVersion of mongodb, we might be in 4.2 but without `startAfter` support. Instead, you can do something like this:

```
db.adminCommand( 
    { 
        getParameter: 1, 
        featureCompatibilityVersion: 1 
    } 
)
```

it will return something like `{ "featureCompatibilityVersion" : { "version" : "4.2" }, "ok" : 1 }`, which will be better to test the compatibility